ReCaptchaFrontendUi/view/frontend/web/js/reCaptcha.js

```diff
@@ -28,7 +28,17 @@ define(
              */
             initialize: function () {
                 this._super();
-                this._loadApi();
+
+                this.attachFocusEvent();
+            },
+
+            attachFocusEvent: function () {
+                let parentForm = $('#' + this.getReCaptchaId() + '-container').parents('form');
+                let self = this;
```

Comment:
An alternative to this variable would be to `bind()` the `this` context into the new function below, or use an arrow function.

```suggestion
                const self = this;
```